ui/src/JobDetails.js

```diff
@@ -219,7 +219,10 @@ class JobDetails extends Component {
         "CANCELLED",
       ].includes(this.props.job.jobStatus) &&
       this.props.job.rowErrorCount > 0 &&
-      this.props.authorisedActivities.includes("LOAD_SAMPLE")
+      (this.props.job.jobType === "SAMPLE" &&
+        this.props.authorisedActivities.includes("LOAD_SAMPLE")) ||
+        (this.props.job.jobType === "BULK_REFUSAL" &&
+          this.props.authorisedActivities.includes("LOAD_BULK_REFUSAL")))
```

Comment:
I would drive this via a prop

for example....

```js
this.props.authorisedActivities.includes(this.props.loadPermission)
```

Then just supply that prop in the component above, for example:
```
  <JobDetails loadPermission="LOAD_SAMPLE">
```